en/mulesoft/modules/ROOT/pages/ms_composer_salesforce_marketing_cloud_reference.adoc

```diff
@@ -0,0 +1,59 @@
+= Salesforce Marketing Cloud Connector Reference
+
+Salesforce Marketing Cloud is a marketing platform that supports capabilities such as real-time customer engagement, email and marketing automation, social media engagement, mobile (SMS) messaging and push notifications, and customer marketing analytics.
+
+== Connections
+
+To connect to a system, whether it is a source of data or the target where you are sending data, you must create a connection with that system's required credentials. You typically connect to different systems within a flow and can reuse each connection every time you need to connect.
+
+For more information on how to create an API integration in Installed Packages, refer to https://developer.salesforce.com/docs/marketing/marketing-cloud/guide/integration-s2s-client-credentials.html[Server-to-Server Integrations with Client Credentials Grant Type].
+
+This system requires the following credential information for its connections:
+
+Connection Display Name::
+
+Enter a connection name that will help you remember the details of this connection.
+You can reuse connections multiple times in a single flow, and in more than one flow,
+and the credentials are hidden after you create the connection. Therefore, it's helpful
+to provide a name that easily identifies this connection from others.
```

Comment:
I would leave out "from others." Otherwise, I suggest "that easily distinguishes this connection from others."